README.md

```diff
@@ -1,19 +1,25 @@
-# Welcome to Remix!
+# Welcome to Plan-it Social
 
-- [Remix Docs](https://remix.run/docs)
+Welcome to `plan-it-social-web`! These are some quick notes on how to start contributing to our open source project. A good place to start is to join our [Discord server](https://discord.gg/tTD7PvwpuX).
 
-## Deployment
+The group meets most Mondays 6pm PST ([meetup](https://www.meetup.com/all-things-web-react-html-css-javascript-tutoring/))
 
-After having run the `create-remix` command and selected "Vercel" as a deployment target, you only need to [import your Git repository](https://vercel.com/new) into Vercel, and it will be deployed.
+## Helpful Links
 
-If you'd like to avoid using a Git repository, you can also deploy the directory by running [Vercel CLI](https://vercel.com/cli):
+- [Discord server](https://discord.gg/tTD7PvwpuX)
+- [Figma wire-frame](https://www.figma.com/file/6e3cBuEHOIpWvqT31Zd29p/Social-Plan-it?type=design&node-id=0-1&mode=design&t=DpLSfRITCDDG1pj0-0)
+- [GitHub](https://github.com/social-plan-it)
+- [Meetup](https://www.meetup.com/all-things-web-react-html-css-javascript-tutoring/)
 
-```sh
-npm i -g vercel
-vercel
-```
+## What we're using
 
-It is generally recommended to use a Git repository, because future commits will then automatically be deployed by Vercel, through its [Git Integration](https://vercel.com/docs/concepts/git).
+- [MySQL](https://www.mysql.com/)
+- [Prisma](https://www.prisma.io/)
+- [Tailwind CSS](https://tailwindcss.com/)
+- [Typescript](https://www.typescriptlang.org/)
+- [React 18](https://react.dev)
```

Comment:
NIT: Just "React" without pointing the verion.

Reply:
I'd like to leave 18, it's a signal to any reviewing our project we're keeping up with the latest and greatest of React. Might be a moot point with our use of Remix.